lib/puppet/provider/templates/webadministration/_getwebsites.ps1.erb

```diff
@@ -22,5 +22,6 @@ Get-WebSite | %{
     logtruncatesize      = [string]$_.LogFile.truncateSize
     loglocaltimerollover = [string]$_.LogFile.localTimeRollover
     logextfileflags      = [string]$_.LogFile.logExtFileFlags
+    preloadenabled       = [string](Get-ItemProperty -Path "IIS:\Sites\$($_.Name)" -Name 'applicationDefaults.preloadEnabled' -ErrorAction 'Continue').Value
```

Comment:
So this looks like the preloadEnabled element should really be defined on the application not at the site level.  Am I missing something here?

Reply:
Also this is setting the default of all applications within this site, whereas shouldn't we be modifying the specific application?

Reply:
Although I can see this gives us parity with the DSC implementation.

Reply:
There are two different settings, application and site. I'm not understanding the comment here

Reply:
I guess my point was.  This is setting the application defaults, but there's corresponding parameter on the iis_application type for preloadenabled.  I guess this is ok so at least the bug is fixed as opposed to a new parameter too.